charts/delphai-deployment/values.yaml

```diff
@@ -159,6 +159,11 @@ ingress:
   # Run Cloudflare purge_cache after deployment
   purgeCloudflareCache: false
 
+# Enable access to OpenID Connect provider
+# When enabled, a new env variable OIDC_BASE_URL will be exposed.
+openidConnect:
+  enabled: false
```

Comment:
Why do disable it? Why not just have it for all deployments?

Reply:
The QRPC pipelines just process RabbitMQ messages and never interact with any JWT tokens.

Reply:
But still having it there won't harm. And we might want to still validate tokens for second-line services. Like for harting for example
I'd just enable it by default instead of adding this key manually to 20 services

Reply:
Conditionals removed in f7424f7